bin/build

```diff
@@ -27,13 +27,15 @@ if [ $DEBUG = true ]; then
   echo "Building secrets-provider-debug Docker image"
 
   docker build \
+    --build-arg TAG="dev" \
     --tag "secrets-provider-for-k8s-debug:latest" \
     --target "secrets-provider-debug" \
     .
 else
   echo "Building secrets-provider-for-k8s:$FULL_VERSION_TAG Docker image"
 
   docker build \
+    --build-arg TAG=$(git_tag) \
```

Comment:
if you want to follow Geri's note then just use short_version_tag found here https://github.com/cyberark/secrets-provider-for-k8s/blob/master/bin/build_utils#L11

Reply:
this will output the following message:
```
CSPFK008I CyberArk Secrets Provider for Kubernetes v1.1.0-1.1.0 starting up
```

which is not what we want. what is this "Geri's note" that you're referring to?

Reply:
See [here](https://conjurhq.slack.com/archives/C2V5T1NRZ/p1605023744116100?thread_ts=1605022517.115600&cid=C2V5T1NRZ)

Reply:
yes, i saw that and referred to it in the other comment. do you still think we should use `short_version_tag` here? i want to first remove the hard-coded `dev` tag and then properly fix it for static versions.

Reply:
but wont we get `1.1.0-`? 
I actually think the commit hash would be nice so we know exactly which commit a customer is working with, not just the version which is much easier to troubleshoot. 
If you agree then tag should be `(git_tag)`

Reply:
closed as in merge the PR? because this PR adds the `git_tag`.

I think that we need the commit hash as long as it is not a released version. official versions should not include the commit hash but that will be taken care of in another issue.